pkg/apis/convert/conversion_helper.go

```diff
@@ -309,3 +325,15 @@ func FromV1beta1SubscribableSpec(from *eventingduckv1beta1.SubscribableSpec) *ev
 	}
 	return &to
 }
+
+// A helper function to mark v1alpha1 Deprecated Condition in the Status.
+// We mark the Condition in status during conversion from a higher version to v1alpha1.
+func MarkV1alpha1Deprecated(cs *apis.ConditionSet, s *pkgduckv1.Status) {
```

Comment:
I'd suggest you also add another comment to this and the remove method saying something like:
`//TODO remove after the 0.17 cut.`